snorkel/parser.py

```diff
@@ -7,45 +7,79 @@
 import json
 import lxml.etree as et
 import os
-import random
 import re
 import requests
 import signal
 from subprocess import Popen
 import sys
 import warnings
 
-from .models import Document, Sentence, construct_stable_id
-from .udf import UDF
-from .utils import ProgressBar, sort_X_on_Y
+from .models import Candidate, Context, Document, Sentence, construct_stable_id
+from .udf import UDF, UDFRunner
+from .utils import sort_X_on_Y
 
 
-class DocParser:
+class CorpusParser(UDFRunner):
```

Comment:
It's not changing any functionality, just a bit of renaming. CorpusParser is actually more like the old SentenceParser, since that's the actual UDF we're distributing.